modules/aws/s3.go

```diff
@@ -62,6 +62,36 @@ func FindS3BucketWithTagE(t testing.TestingT, awsRegion string, key string, valu
 	return "", nil
 }
 
+// GetS3BucketTagging fetches the given bucket's tags and returns them as a string map of strings.
+func GetS3BucketTagging(t testing.TestingT, awsRegion string, bucket string) map[string]string {
```

Comment:
Let's go with `GetS3BucketTags`, so it's consistent with the other terratest resources.